### ISSUE TITLE
[BUG] LibDBIcon path wrong or missing

### ISSUE THREAD

**Issue by Hayato2846** [PhenomDevel/BlizzHUDTweaks#70]
## Description
> 2x BlizzHUDTweaks/BlizzHUDTweaks.toc:24 Couldn't open BlizzHUDTweaks/Libs/LibDBIcon/LibDBIcon-1.0-52/lib.xml

## Reproduce
> What steps are needed to reproduce it?
![image](https://github.com/user-attachments/assets/04d51e4e-7c5d-44f0-99cb-511b80290768)

## Expected Behaviour
> Guess it's just an oversight of updating the library

## Optional Context (e.g. Screenshots)

Cheers (:

**Comment by PhenomDevel** [comment]
Do you have any other addon which includes LibDBIcon? 

I checked it on my local machine and do not have this behaviour. Nor do i have set the path to `BlizzHUDTweaks/Libs/LibDBIcon/LibDBIcon-1.0-52/lib.xml`.

**Comment by Hayato2846** [comment]
With only BlizzHUDTweaks enabled (beside BugGrabber and BugSack) I still get following two errors:

But I checked the toc file and it is set properly for me. Updated via WowUp / CurseForge. 

```
1x BlizzHUDTweaks/BlizzHUDTweaks.lua:14: Cannot find a library instance of "LibDBIcon-1.0".
[string "=[C]"]: in function `error'
[string "@BlizzHUDTweaks/Libs/LibStub/LibStub.lua"]:38: in function `LibStub'
[string "@BlizzHUDTweaks/BlizzHUDTweaks.lua"]:14: in main chunk

Locals:
(*temporary) = "Cannot find a library instance of "LibDBIcon-1.0"."
```

`2x BlizzHUDTweaks/BlizzHUDTweaks.toc:24 Couldn't open BlizzHUDTweaks/Libs/LibDBIcon/LibDBIcon-1.0-52/lib.xml`

**Comment by durrburger93** [comment]
I'm getting the same issue as Hayato, only Blizzhudtweaks is enabled and I'm seeing the same error. The addon isn't working and there's nothing in the addons section of options/no minimap icon.

**Comment by PhenomDevel** [comment]
Currently I'm a bit confused, because the package that is available on Github as a release looks okay and should be 1:1 the same as what I use locally for testing.
I'll download the package later via CurseForge and see if I have the same problem. Alternatively, I'll roll back the version of LibDBIcon for now.

**Comment by Arkewas** [comment]
I have 1.45.1 version and I get the same error as above (13x BlizzHUDTweaks/BlizzHUDTweaks.toc:24 Couldn't open BlizzHUDTweaks/Libs/LibDBIcon/LibDBIcon-1.0-52/lib.xml )
BlizzHUDTweaks seems to work fine btw.

**Comment by PhenomDevel** [comment]
`local LibDBIcon = LibStub("LibDBIcon-1.0")` should be the correct way of loading the library. The minor version should be irrelevant for loading the library. Some errors mention the minor version when trying to load the library. That is something strange i don't really get. 

For now i'll just rollback the change so the addon will work for now since the change was made because i thought some problem came from the library. As it turned it the problem was just a deprecated wow ui function. So i believe i can fix that problem without having to upgrade the lib.

**Comment by PhenomDevel** [comment]
🤦 I think i found the problem... Since i updated the library i changed the `path` pointing to the `lib.xml`... In my local env it is fine and correct because the folder structure changed.

For the build package you guys download it is not since the libraries are loaded on build time and package differently to my local machine. So for the github build the paths do not match....

Sry :D

**Comment by Arkewas** [comment]
> So the addon still works fine even though you get the error? Do you still have a minimap icon?

Yes, I have BlizzHudTweaks minimap icon and also I have BlizzHudTweaks icon in Titan Panel addon. The minimap icon was hidden so I used the command /bth minimap to show it again.

**Comment by PhenomDevel** [comment]
@Arkewas @Hayato2846 @durrburger93 I created 1.45.2. Please check if this fixes the problem.

**Comment by Arkewas** [comment]
With 1.45.2 I don't get the error anymore and BlizzHUDTweaks is working fine. I also have both minimap icon and Titan Panel icon. Thank you for fixing the issue.

**Comment by Hayato2846** [comment]
@PhenomDevel Sorry for the late answer. Yes it's fixed with 1.45.2 :) Thanks!
